lua/luasnip/nodes/snippet.lua

```diff
@@ -185,6 +185,11 @@ local function init_snippet_context(context)
 	-- might be nil, but whitelisted in snippetProxy.
 	context.priority = context.priority
 
+	-- keep value, set to nil to signal that this option is unset technically
+	-- this can be removed but having this here to state the options of the
+	-- context-table is nice
+	context.autotriggered = context.autotriggered or nil
```

Comment:
This might backfire if `context.autotriggered` is set to `false` :D
You make a good point in the comment though, maybe just omit the `or nil` like above, and put a comment on which values are fine.

Reply:
Well yes this was a mistake. Should be fixed by now